pkg/apis/helm.fluxcd.io/v1/types.go

```diff
@@ -18,6 +18,11 @@ type LocalObjectReference struct {
 	Name string `json:"name"`
 }
 
+type ObjectReference struct {
```

Comment:
```suggestion
type NamespacedObjectReference struct {
```

may be more fitting.

Reply:
Never mind, looking at the Kubernetes source `ObjectReference` is fine as well.